PowerEditor/src/ScitillaComponent/AutoCompletion.cpp

```diff
@@ -43,7 +43,11 @@ static bool isInList(generic_string word, const vector<generic_string> & wordArr
 
 static bool isAllDigits(const generic_string &str)
 {
+#ifdef _UNICODE

```

Comment:
Notepad++ is Unicode. ANSI version was dropped long back. Moreover, ```generic_string``` is ```typedef std::basic_string<TCHAR> generic_string;``` where ```TCHAR``` is ```WCHAR``` because of Unicode project setting. Also refer discussion - https://github.com/notepad-plus-plus/notepad-plus-plus/pull/3198#discussion_r113758228

Reply:
Thank you for reviewing. I will remove that compiler encounter.

Reply:
Then, Can I erase the compiler encounters without considering ASCII situations? And was there an issue in the context of using iswdigit?

Reply:
> Notepad++ is Unicode. ANSI version was dropped long back. Moreover, `generic_string` is `typedef std::basic_string<TCHAR> generic_string;` where `TCHAR` is `WCHAR` because of Unicode project setting. Also refer discussion - [#3198 (comment)](https://github.com/notepad-plus-plus/notepad-plus-plus/pull/3198#discussion_r113758228)

Then why not replace all references of **T** in the entire project with **W**. Add some ``static_assert`` or `#define` all T-names to something that would cause a compiler error, and preventing T-uses.